### DISCUSSION TITLE
Getting no chia

### DISCUSSION THREAD

**Discussion by Mutwy** [Chia-Network/chia-blockchain#9496] [Support]
### Discussed in https://github.com/Chia-Network/chia-blockchain/discussions/9489

<div type='discussions-op-text'>

<sup>Originally posted by **Mutwy** December  6, 2021</sup>
Hello miners,

I'm experiencing a big issue with Chia latelly and I'm pretty worried about my account.
I have 1.6 PB of plots and it's been almost a month, I think, from the last time I got a Chia on my account.
I'm not attached to any pool.
I think this started after  from update 1.2.9

Could you help me figure out the issue.
Regards,
Mat</div>

**Comment by RWilsonChia** [comment]
Issue migrated to support - as this is a support issue.

**Comment by GuyCre8ive** [comment]
I just hit a PB and haven't hit solo for over a month.

**Reply by Mutwy** [reply]
Any idea why this is happening?
It's so strange to have "time to win" at 5 days but win absolutely nothing in over a month....

**Reply by GuyCre8ive** [reply]
I have no idea. Early on, one of the first farmers told me they saw 7x their ETTW so maybe we're both just seeing that. Someone else on reddit said they're in the same boat though: https://www.reddit.com/r/chia/comments/raigid/comment/hnj17s8/?utm_source=share&utm_medium=web2x&context=3

**Reply by GuyCre8ive** [reply]
I hope the developers will let us know if they did something to favor pooling over solo, the guy on reddit is saying his pooling plots are normal. I'm running the current build on a couple Server 2019 rigs with NLB and I'm skipping the passphrase. I have Frontier as my ISP but I doubt that would matter.

**Reply by Mutwy** [reply]
I don't think it matters as well, I have 2 linux machines with way too many disks :D 
Will see what happens in near future...
Did you think about converting to public pools instead of solo?

**Reply by GuyCre8ive** [reply]
I'm gonna let it go another month solo and if I still don't hit, I'll probably just give up entirely. I was hoping to use Chia as a global layer for my project but not winning a block in 2 months with what I have invested in this will make me lose my faith in it.

**Reply by Mutwy** [reply]
Same here... way too much time and money invested in this project.
